simpl/config.py

```diff
@@ -438,6 +438,8 @@ def __len__(self):
 
     def __getattr__(self, attr):
         """Get attribute."""
+        if attr == '_values':
+            raise AttributeError()
```

Comment:
Wow. This was an intricate bug. Is it worth documenting why we're doing this?

```
        # avoid infinite recursion if ._values doesn't exist which can happen when __init__
        # is skipped during deserialization or certain types of copy operations
        # Ref: the URL
```

Or in the commit message.